lib/bots/syncbot.ts

```diff
@@ -142,7 +141,10 @@ export class SyncBot extends ProcBot {
     private handleError(error: Error, event: InterimContext): void {
         // Put this on the log service
         this.logger.log(LogLevel.WARN, error.message);
-        this.logger.log(LogLevel.WARN, JSON.stringify(event));
+        const secrets = [];
+        if (event.toIds.token) { secrets.push(event.toIds.token); }
+        if (event.sourceIds.token) { secrets.push(event.sourceIds.token); }
+        this.logger.log(LogLevel.WARN, JSON.stringify(event), secrets);
```

Comment:
Maybe:
```typescript
this.logger.log(LogLevel.WARN, JSON.stringify(event),
    _.filter(_.map([event.toIds, event.sourceIds], (thing) => thing.token), (token) => token)));
```
instead of the two ifs would be better, or break the test out into a static method, as it's used by both handlers?
If we're keeping the `ifs`, can we please follow convention in the rest of the framework and do:
```typescript
if () {
    statement;
}```